### PR TITLE
issue/2490 - Identify the Affiliate Area page in the list table screen

### PR DESCRIPTION
**Pull request by tubiz**
Issue: #2490

### CODE REVIEW

File: affiliate-wp.php [hunk 1]
```diff
@@ -382,6 +382,7 @@ private function includes() {
 			require_once AFFILIATEWP_PLUGIN_DIR . 'includes/admin/tools/class-migrate.php';
 			require_once AFFILIATEWP_PLUGIN_DIR . 'includes/admin/add-ons.php';
 			require_once AFFILIATEWP_PLUGIN_DIR . 'includes/admin/user-profile.php';
+			require_once AFFILIATEWP_PLUGIN_DIR . 'includes/admin/posts-list.php';
```
Comment (DrewAPicture) [inline affiliate-wp.php]: Rather than introduce a new top-level admin file just for this, let's put this code in includes/actions.php.

**Comment by tubiz** [comment]
The requested changes has been made.

File: includes/actions.php [hunk 2]
```diff
@@ -93,3 +93,22 @@ function affwp_process_add_affiliate_website( $affiliate_id, $args ) {
 
 }
 add_action( 'affwp_insert_affiliate', 'affwp_process_add_affiliate_website', 11, 2 );
+
+/**
+ * Add a post display state for AffiliateWP pages in the page list table.
```
Comment (DrewAPicture) [inline includes/actions.php]: Minor nitpick here, we need to use third-person singular verbs in DocBlock summaries. In this case, that would be "Adds". Everything else looks great.

Comment (tubiz) [reply includes/actions.php]: Updated. Thanks for pointing it out.

**Pull request merged**